ci/nosetests.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -u
+
+EXIT_CODE=0
+
+echo "========================================="
+echo "nosetests -a '!requires_sudo' output"
+echo "========================================="
+nosetests -a '!requires_sudo' tests $NOSEOPTS || EXIT_CODE=$(($EXIT_CODE || $?))
+echo EXIT_CODE no_sudo nosetests = $EXIT_CODE
+sudo mv .coverage coverage.$(uuidgen).no_sudo.data
```

Comment:
Do you still need these uuids?

Reply:
Yes, to combine the .coverage files for codecov.

Reply:
I see, thanks